compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -276,12 +268,67 @@ void addGPUMatmulTensorCorePassPipeline(OpPassManager &pm,
   // Hoist loop invariant code to avoid pipelining it.
   nestedModulePM.addNestedPass<func::FuncOp>(
       createLoopInvariantCodeMotionPass());
-  PipeliningSchedulingStrategy schedule =
-      llvmgpuUseMMASync ? PipeliningSchedulingStrategy::nvidiaTensorCore
-                        : PipeliningSchedulingStrategy::loadGlobalStage0;
   // Pipeline memory operations.
   nestedModulePM.addNestedPass<func::FuncOp>(createGPUPipeliningPass(
-      /*epiloguePeeling=*/false, pipelineDepth, schedule));
+      /*epiloguePeeling=*/false, pipelineDepth,
+      PipeliningSchedulingStrategy::loadGlobalStage0));
+}
+
+void addGPUMatmulTensorCoreMmaSyncPassPipeline(OpPassManager &pm,
```

Comment:
Maybe we can add comment for the function header that this pass pipeline is for native support of NVIDIA Tensor Cores. This is specific to NVIDIA with MmaSync in the name.